### PR TITLE
PLAT-140367: WizardPanels: Fix touch effect is shown at another button

### PR DESCRIPTION
**Pull request by juwonjeong**
Enact-DCO-1.0-Signed-off-by: Juwon Jeong (juwon.jeong@lge.com)

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Touch effect is shown at another button in the footer of WizardPanels when navigating panels.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
This issue was solved in https://github.com/enactjs/sandstone/pull/871.
This PR is for applying the patch to develop.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
PLAT-140367

### CODE REVIEW

File: samples/sampler/stories/qa/WizardPanels.js [hunk 1]
```diff
@@ -6,7 +6,9 @@ import Button from '@enact/sandstone/Button';
 import Icon from '@enact/sandstone/Icon';
 import Item from '@enact/sandstone/Item';
 import {Scroller} from '@enact/sandstone/Scroller';
+import {Panel} from '@enact/sandstone/WizardPanels';
 import WizardPanels, {WizardPanelsBase} from '@enact/sandstone/WizardPanels';
```
Comment (jeonghee27) [inline samples/sampler/stories/qa/WizardPanels.js:10]:
```suggestion
import WizardPanels, {Panel, WizardPanelsBase} from '@enact/sandstone/WizardPanels';
```
I

File: CHANGELOG.md [hunk 2]
```diff
@@ -15,6 +15,7 @@ The following is a curated list of changes in the Enact sandstone module, newest
 
 ### Fixed
 
+- `sandstone/WizardPanels` to not show focus effect on wrong element in `footer`
```
[the remark this answers is not in the document]
Comment (juwonjeong) [reply CHANGELOG.md]: It's fixed.

File: samples/sampler/stories/qa/WizardPanels.js (hunk 1 above)
Comment (juwonjeong) [reply samples/sampler/stories/qa/WizardPanels.js]: It's fixed.

**Review by jeonghee27 — APPROVED** [review]
LGTM

File: samples/sampler/stories/qa/WizardPanels.js [hunk 3]
```diff
@@ -24,6 +25,47 @@ const inputData = {
...
+
+	onPrevClick = (ev) => {
+		this.setState({index: 0});
+		action('onPrevClick')(ev);
+	};
+
+	render () {
+		return (
+			<WizardPanels
+				noAnimation index={this.state.index}
```
Comment (MikyungKim) [inline samples/sampler/stories/qa/WizardPanels.js:49]:
```suggestion
				noAnimation
				index={this.state.index}
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -20,6 +20,7 @@ The following is a curated list of changes in the Enact sandstone module, newest
 - `sandstone/Input` button label when default value is `0`
 - `sandstone/Scroller` and `sandstone/VirtualList` to activate voice control intent when only scrollable
 - `sandstone/VideoPlayer` to show the knob when mediaSlider gets focused with 5-way
+- `sandstone/WizardPanels` to not show focus effect on wrong element in `footer`
```
Comment (MikyungKim) [inline CHANGELOG.md:23]:
```suggestion
- `sandstone/WizardPanels` to not show focus effect on the wrong element in `footer` when touch
```

File: samples/sampler/stories/qa/WizardPanels.js (hunk 3 above)
Comment (juwonjeong) [reply samples/sampler/stories/qa/WizardPanels.js]: It's fixed.

File: CHANGELOG.md (hunk 4 above)
Comment (juwonjeong) [reply CHANGELOG.md]: It's fixed.

**Review by MikyungKim — APPROVED** [review]
💯 🎉

**Comment by seunghoh** [comment]
㊗️

**Pull request merged**